core/templates/dev/head/services/SpeechSynthesisChunkerService.ts

```diff
@@ -170,21 +170,22 @@ export class SpeechSynthesisChunkerService {
     var elt = $('<div>' + html + '</div>');
     // Convert links into speakable text by extracting the readable value.
     elt.find('oppia-noninteractive-' + this.RTE_COMPONENT_NAMES.Link)
-      .replaceWith(() => {
+      .replaceWith(function() {
```

Comment:
Arrow function was using lexically `this` so changed it to a `function` such that it uses the `elt` this object.

Reply:
Looks good then, LGTM!